### ISSUE TITLE
LibreTube is unable to play videos with channel names ending with "Topic"

### ISSUE THREAD

**Issue by gaining** [libre-tube/LibreTube#5872]
### Steps to reproduce

Play a video from a channel name ending in "topic" and it will say "Source error" and probably crash. I use this [playlist](https://www.youtube.com/playlist?list=PLN4MwbWkCSmXFehtW0zhcqhRgE6Dtjg2b) and it won't play any of the videos from channels that ends with "topic" however it will play all the other ones

### Expected behavior

It used to play them just fine, but it stopped playing it a week ago I think.

### Actual behavior

"source error" error message

### LibreTube version

0.22.2

### Android version

Android 13

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I will fill out all of the requested information in this form.

**Comment by Bnyro** [comment]
I can't reproduce this, likely the instance you're using is partially blocked by YouTube.
